var/spack/repos/builtin/packages/bcl2fastq2/package.py

```diff
@@ -32,18 +31,23 @@ class Bcl2fastq2(Package):
 
     conflicts("platform=darwin", msg="malloc.h/etc requirements break build on macs")
 
-    depends_on("boost@1.54.0:1.55")
+    # Pulled from the fallback vendored-build of boost
+    # TODO: supporting boost > 1.54 will require patching bcl2fastq/src/cxx/lib/io/Xml.cpp
+    # as described in https://gist.github.com/jblachly/f8dc0f328d66659d9ee005548a5a2d2e
+    depends_on("boost@1.54")
```

Comment:
Not sure if this will make a difference but I used `boost@1.54.0` instead to ensure the patch number of 0 is used as per the redist tarball (in my patch, as well as making this package statically linked.  See https://github.com/spack/spack/pull/48658 )